tests/robustness/multiclient_test/framework/harness.go

```diff
@@ -145,6 +145,13 @@ func (th *TestHarness) getSnapshotter() bool {
 		return false
 	}
 
+	contentCacheLimitMB := "--content-cache-size-limit-mb=500"
+	metadataCacheLimitMB := "--metadata-cache-size-limit-mb=500"
+	if err = s.SetCacheSizeLimits(contentCacheLimitMB + metadataCacheLimitMB); err != nil {
```

Comment:
Does this work? it seems to me that this will always return an error.

Reply:
Updated the code